cli/lib/kontena/scripts/completer

```diff
@@ -3,238 +3,7 @@
 
 # resolve bin path, ignoring symlinks
 require 'pathname'
-require 'yaml'
```

Comment:
Is `yaml` available on `kontena/scripts/completer.rb`? Or is this totally redundant require?